Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ReflectedPropertyEditor.hxx

```diff
@@ -79,7 +79,8 @@ namespace AzToolsFramework
 
         void SetFilterString(AZStd::string str) override;
         AZStd::string GetFilterString();
-        void SetSavedStateKey(AZ::u32 key) override; // a settings key which is used to store and load the set of things that are expanded or not and other settings
+        // a settings key which is used to store and load the set of things that are expanded or not and other settings
+        void SetSavedStateKey([[maybe_unused]] AZ::u32 key, [[maybe_unused]] AZStd::string propertyEditorName = "") override;
```

Comment:
A better default value for the AZStd::string is AZStd::string{}.
That way a `strlen("")` call is saved. The `AZStd::string(const char*)` constructor has to perform a string length to determine the size of a c-string, which for an empty string is 0.
So it is better to just use the AZStd::string default constructor(This applies to any string like class by the way).
```suggestion
        void SetSavedStateKey([[maybe_unused]] AZ::u32 key, [[maybe_unused]] AZStd::string propertyEditorName = AZStd::string{}) override;
```

Reply:
This is awesome, thank you. Fixed here and other places.